### PR TITLE
Add CSV parser

### PR DESCRIPTION
**Pull request by fmrsabino**
Part of #81

- Introduces a utility function `csv_helper.parse` to parse CSV files.
- This utility function takes a file stream and yields a tuple representing the row items

### CODE REVIEW

File: safe_locking_service/utils/csv_helper.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import csv
+
+
+def parse(stream, skip_header: bool = True):
```
Comment (moisses89) [inline safe_locking_service/utils/csv_helper.py:4]: Could you define the stream type? Like stream : <anytype>?

**Review by moisses89 — APPROVED** [review]
Looks good but just a minor change.

File: safe_locking_service/utils/csv_helper.py (hunk 1 above)
Comment (fmrsabino) [reply safe_locking_service/utils/csv_helper.py]: Done in [6732dec](https://github.com/safe-global/safe-locking-service/pull/89/commits/6732decfb51ab918244f0a6a9736c6725a791a86)

File: safe_locking_service/utils/csv_helper.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import csv
+from typing import IO
+
+
+def parse(stream: IO, skip_header: bool = True):
```
Comment (Uxio0) [inline safe_locking_service/utils/csv_helper.py:5]: Missing return type

File: safe_locking_service/utils/tests/test_csv.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+from .. import csv_helper
+
+
+class CSVTestCase(unittest.TestCase):
+    def test_parsing(self):
+        csv_content = "name,age\nJohn,30\nJane,25"
+        with StringIO(csv_content) as file:
+            result = list(csv_helper.parse(file))
+
+            assert result == [["John", "30"], ["Jane", "25"]]
```
Comment (Uxio0) [inline safe_locking_service/utils/tests/test_csv.py:13]: IMHO you should use `TestCase` assert methods: https://stackoverflow.com/a/6361186/724991

Comment (fmrsabino) [reply safe_locking_service/utils/tests/test_csv.py]: Agreed! Adjusted in [341972f](https://github.com/safe-global/safe-locking-service/pull/89/commits/341972f93c9e0e9e67530f4c2113f3fa5aa50838)

**Review by Uxio0 — APPROVED** [review]

File: safe_locking_service/utils/tests/test_csv.py [hunk 4]
```diff
@@ -10,18 +10,18 @@ def test_parsing(self):
         with StringIO(csv_content) as file:
             result = list(csv_helper.parse(file))
 
-            assert result == [["John", "30"], ["Jane", "25"]]
+            self.assertEqual(result, [["John", "30"], ["Jane", "25"]])
```
[the remark this answers is not in the document]
Comment (fmrsabino) [reply safe_locking_service/utils/tests/test_csv.py:13]: Do you think we should change it? According to the docs:

> These methods are used by default when comparing lists or tuples with assertEqual

I think the advantage of `assertEqual` here is that the correct function will be called if the types change (e.g. with a refactor) https://docs.python.org/3.12/library/unittest.html#unittest.TestCase.assertListEqual

[the remark this answers is not in the document]
Comment (Uxio0) [reply safe_locking_service/utils/tests/test_csv.py:13]: Oh ok! I see it was changed recently

**Pull request merged**